lib/action_cable/server/base.rb

```diff
@@ -36,11 +36,11 @@ def worker_pool
         @worker_pool ||= ActionCable::Server::Worker.pool(size: config.worker_pool_size)
       end
 
-      # Requires and returns an array of all the channel class constants in this application.
-      def channel_classes
+      # Requires and returns an hash of all the channel class constants in this application.
+      def channel_classes_hash
```

Comment:
Let's just keep the name `channel_classes`. It's fine to have that return a hash instead of an array.